### ISSUE TITLE
feat: data property support pass string array to hightlight

### ISSUE THREAD

**Issue by AngryChocobo** [roy-kang/text-marker#1]
If I want to highlight some words in article this library cannot work. It need word position info so user must `mark` them manually am I right?

**Comment by roy-kang** [comment]
Yes, that's the situation now. I'll see if I can add that feature later.
